### ISSUE TITLE
hello frakman1  May I ask if you are still doing openvas related work recently?

### ISSUE THREAD

**Issue by JohnWangggg** [frakman1/frakman1#1]
We're trying to make Nessus NASL scripts compatible with OpenVAS. I was wondering how your progress has been so far and if you have any experiences you could share with us?

**Comment by JohnWangggg** [comment]
@frakman1

**Comment by frakman1** [comment]
No, sorry. I don't have any update to report and am not working on that anymore.

Good luck
